sdk/account-info/src/lib.rs

```diff
@@ -28,7 +29,7 @@ pub struct AccountInfo<'a> {
     /// Program that owns this account
     pub owner: &'a Pubkey,
     /// The epoch at which this account will next owe rent
-    pub rent_epoch: Epoch,
+    pub rent_epoch: u64,
```

Comment:
Pubkey is a struct, and not a type, so it would be a breaking change to use anything else.

I might be missing something, but what's the reasoning to prefer the `Epoch` type alias? The idea was to avoid an extra dependency.

Reply:
> Ah, that part makes sense. How "expensive" is it to have this extra dependency?

Maybe @kevinheavey has some numbers on that, or I can whip them up

> Since Epoch is a type alias, we lose some of that, but we still retain search and the self-documenting aspects of Epoch.

Gotcha, I can see from the documenting aspect that it could be useful. My thinking was that:

* I've never seen the rent epoch used by a program
* it introduces another dependency
* since it's just a type alias, it doesn't really matter

For those reasons, I lean towards keeping it as is, but if you feel strongly about it, we can change it.

Reply:
Sounds good! And thanks for clarifying the difference between type and type alias :sweat_smile:

Reply:
yeah on my machine adding the solana-clock dependency increases solana-account-info build time by 40%. It's made quite expensive by the proc macro deps in solana-clock